resources/recursor_config.rb

```diff
@@ -44,7 +44,7 @@
 
 property :source, [String, nil], default: 'recursor_service.conf.erb'
```

Comment:
why do these take nil and what happens if I set `source nil` or `cookbook nil` in my wrapper instead of defaults? I don't understand the usecase